website/content/en/v0.10.0/getting-started/migrating-from-cas/scripts/step07-tag-security-groups.sh

```diff
@@ -5,10 +5,20 @@ LAUNCH_TEMPLATE=$(aws eks describe-nodegroup --cluster-name ${CLUSTER_NAME} \
     --nodegroup-name ${NODEGROUP} --query 'nodegroup.launchTemplate.{id:id,version:version}' \
     --output text | tr -s "\t" ",")
 
-SECURITY_GROUPS=$(aws ec2 describe-launch-template-versions \
+if [ `aws ec2 describe-launch-template-versions \
     --launch-template-id ${LAUNCH_TEMPLATE%,*} --versions ${LAUNCH_TEMPLATE#*,} \
     --query 'LaunchTemplateVersions[0].LaunchTemplateData.SecurityGroupIds' \
-    --output text)
+    --output text` == 'None' ]; then
+    SECURITY_GROUPS=$(aws ec2 describe-launch-template-versions \
+        --launch-template-id ${LAUNCH_TEMPLATE%,*} --versions ${LAUNCH_TEMPLATE#*,} \
+        --query 'LaunchTemplateVersions[0].LaunchTemplateData.NetworkInterfaces[0].Groups' \
+        --output text)
+    else
+    SECURITY_GROUPS=$(aws ec2 describe-launch-template-versions \
+        --launch-template-id ${LAUNCH_TEMPLATE%,*} --versions ${LAUNCH_TEMPLATE#*,} \
+        --query 'LaunchTemplateVersions[0].LaunchTemplateData.SecurityGroupIds' \
+        --output text)
+fi
```

Comment:
You could do this instead. If neither NetworkInterfaces.Groups or SecurityGroupIds are there, then it would be None. This will ensure we check both places and reduces bash complexity.
```
SECURITY_GROUPS=$(aws ec2 describe-launch-template-versions \
    --launch-template-id ${LAUNCH_TEMPLATE%,*} --versions ${LAUNCH_TEMPLATE#*,} \
    --query 'LaunchTemplateVersions[0].LaunchTemplateData.[NetworkInterfaces[0].Groups||SecurityGroupIds]' \
    --output text)
```